QuowMinimap.lua

```diff
@@ -14430,7 +14433,7 @@ function TriggersShowHide()
     -- You defend an enemy special
     SetTriggerOption(tostring("QuowSpecialFailIncCat" .. iN), "omit_from_output", sUSER_OPTIONS["omit_special_inc_fail"])
     -- Enemy hits you with a special
-    SetTriggerOption(tostring("QuowSpecialSuccessIncCat" .. iN), "omit_from_output", sUSER_OPTIONS["omit_col_special_inc_success"])
+    SetTriggerOption(tostring("QuowSpecialSuccessIncCat" .. iN), "omit_from_output", sUSER_OPTIONS["omit_special_inc_success"])
```

Comment:
This corrects a typo in QuowMinimap. `omit_col_special_inc_success` is not a valid key of `sUSER_OPTIONS`, so `sUSER_OPTIONS["omit_col_special_inc_success"]) == nil`.